CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- Remove short options -s on hostname_sub, it was already taken
```

Comment:
Lets put this under a section with `### Breaking Changes` to be nice and loud about this. Also lets indicate which checks this affects.